### DISCUSSION TITLE
Calendar UI Clutter

### DISCUSSION THREAD

**Discussion by Liquidmantis** [nvim-telekasten/telekasten.nvim#42] [Q&A] [answered]
I noticed in your screenshots that your calendar sidebar is dechromed.  I was having issues with things like numbers, relativenumbers, and signcolumn showing up, which breaks the calendar display (I didn't even realize weeks were being displayed!).

I noticed the calendar is setting a filetype, so I just set an after/ftplugin that turns those off, and had to add an exclusion for my indent visualization extension.  

I'm curious, how are your streamlining yours?  I was going to submit a documentation PR with the steps I took if that would be helpful.

**Comment by renerocksai** [comment, marked as answer]
I'd say my calendar sidebar is not dechromed, it is just not chromed. I have `set number` in my `init.vim` but they don't show up in the calendar. The reason for that can be found in the calendar plugin:

![image](https://user-images.githubusercontent.com/30892199/145878853-9810fae1-40b3-4cf5-94a5-a4b3ad67ff6d.png)

Which is set in there like this:
![image](https://user-images.githubusercontent.com/30892199/145878951-52d5a3fb-bc01-466c-9a22-057869fab729.png)

So, the default is: nonumber, norelativenumber, foldcolumn=0. You can check with `:let g:calendar_options`.

I think some other plugin is messing with the calendar.

Please, if you have any tips that would help others with similar issues, make a doc PR like you suggested and we'll take it from there.

**Reply by Liquidmantis** [reply]
Huh, I get the first but it's not picking up the relative numbers and turning it off.  
<img width="316" alt="image" src="https://user-images.githubusercontent.com/399056/145889574-572b49f7-def7-48bb-b951-7e4c0956a28c.png">

I have to turn off relative numbers and signcolumn.  I guess I was wrong about absolute numbers being on.  It's just showing the relnu 0 on current line.  If it's working correctly for others I'll just set mine locally.
